galeracluster/source/mysqlwsrepoptions.rst

```diff
@@ -1103,6 +1103,15 @@ Defines whether the node uses transparent handling of preordered replication eve
 
 This parameter enables transparent handling or preordered replication events, such as replication from a traditional master node. 
 
+Preordered events are not meant to interfere with events that originate on the local node.  That is, they should not update or insert on the same table, because they are applied after replication.  The node eliminates replication latency wait between events rather than first applying and tehn replicating, which limits the slave processing rate to:
```

Comment:
Preordered events should only be used when they will not interfere with update queries that are executed on the cluster itself.

The slave processing rate is limited when the option is disabled. The formula does not apply when the option is enabled.